File: csharp/entity/ScheduleEntity.cs
// Schedule entity client for the Thesmsworks SDK.

using Voxgig.Struct;

namespace ThesmsworksSdk.Entity;

public class ScheduleEntity : ThesmsworksEntityBase
{
    public ScheduleEntity(ThesmsworksSDK client, Dictionary<string, object?>? entopts = null)
        : base(client, entopts, "schedule")
    {
    }

    public override IEntity Make()
    {
        return new ScheduleEntity(client, CloneOpts());
    }

    // (load not defined by this API - base class throws UnsupportedOp)

    // (list not defined by this API - base class throws UnsupportedOp)

    // (create not defined by this API - base class throws UnsupportedOp)

    // (update not defined by this API - base class throws UnsupportedOp)

    // (remove not defined by this API - base class throws UnsupportedOp)
}

File: swift/Sources/ThesmsworksSdk/entity/ScheduleEntity.swift
// Schedule entity client for the Thesmsworks SDK.

import Foundation

public final class ScheduleEntity: ThesmsworksEntityBase {
  public init(_ client: ThesmsworksSDK, _ entopts: VMap? = nil) {
    super.init(client, entopts, "schedule")
  }

  public override func make() -> Entity {
    return ScheduleEntity(client, cloneOpts())
  }

  // (load not defined by this API - base class throws unsupportedOp)

  // (list not defined by this API - base class throws unsupportedOp)

  // (create not defined by this API - base class throws unsupportedOp)

  // (update not defined by this API - base class throws unsupportedOp)

  // (remove not defined by this API - base class throws unsupportedOp)
}
